frontend/ui/src/components/notes/index.tsx

```
@@ -0,0 +1,216 @@
+                    <Pushpin
+                      className={"h-6 w-6 cursor-pointer stroke-gray-12"}
+                    />
+                  </IconButton>
+                  <svg
+                    width="350"
+                    height="2"
+                    viewBox="0 0 350 2"
+                    className="absolute bottom-0 left-0 w-full stroke-gray-7"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)